src/test/resources/Facebook/FacebookLogin.feature

```diff
@@ -0,0 +1,31 @@
+Feature: FacebookLogin
```

Comment:
You should use some tag at Line 1 like @regression or @smoke or something meaningful so that we can run a group of test together.